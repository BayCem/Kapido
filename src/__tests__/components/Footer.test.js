import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Footer from '../../components/footerComponent/Footer';

configure({ adapter: new Adapter() });
it('renders coorectly', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
