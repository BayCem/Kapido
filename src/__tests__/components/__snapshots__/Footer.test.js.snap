// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders coorectly 1`] = `
<footer
  className="Footer"
>
  <nav
    className="Footer-menu"
  >
    <a
      className="Footer-menu__item"
      href="#"
      target="_blank"
    >
      Hakkımızda
    </a>
    <a
      className="Footer-menu__item"
      href="#"
      target="_blank"
    >
      İletişim
    </a>
    <a
      className="Footer-menu__item"
      href="#"
      target="_blank"
    >
      Kariyer
    </a>
  </nav>
</footer>
`;
